docs/tfengine/template_dev.md

```diff
@@ -78,17 +78,78 @@ developers.
     }
     ```
 
-1. Data blocks in templates follow a Tree structure. Values in the `data` block
-    from an upper level template are passed down to its child templates and made
-    available. There is no need to repeat those values in the `data` block in
-    child templates unless you would like to override them. For example, all
-    values specified in the top level template
-    [here](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/dd2464e1a293b0bc549ef34d0c73787e798e88ba/examples/tfengine/team.hcl#L17)
-    are passed down and made available to child templates
-    [root.hcl](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/dd2464e1a293b0bc549ef34d0c73787e798e88ba/examples/tfengine/modules/root.hcl),
-    [foundation.hcl](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/dd2464e1a293b0bc549ef34d0c73787e798e88ba/examples/tfengine/modules/foundation.hcl)
+1. `data` maps can be specified either
+    [inside](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/c27ddc3f1b629f77a70e7b53908882a7e4c94f42/examples/tfengine/team.hcl#L17)
+    or
+    [outside](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/c27ddc3f1b629f77a70e7b53908882a7e4c94f42/examples/tfengine/modules/team.hcl#L15)
+    a `template` block.
+
+    In both cases, values in the `data` maps from an upper level template are
+    passed down to its child template and made available. There is no need to
+    repeat data values in child templates unless you would like to override
+    them. For example, all data values specified in the top level template
+    [here](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/c27ddc3f1b629f77a70e7b53908882a7e4c94f42/examples/tfengine/team.hcl#L17)
+    are passed down and made available to its child templates
```

Comment:
Nit: Mention "its transitive child templates" since team.hcl doesn't directly call foundation.hcl and team.hcl.